api/db/db_models.py

```
@@ -317,6 +404,74 @@ def begin(self):
+        super().__init__(*args, **kwargs)
+
+    def _prepare_sql_for_execution(self, sql):
+        return sql
+
+    def execute_sql(self, sql, params=None, commit=True):
+        for attempt in range(self.max_retries + 1):
+            try:
+                prepared_sql = self._prepare_sql_for_execution(sql)
+                return super().execute_sql(prepared_sql, params, commit)
+            except (OperationalError, InterfaceError) as e:
+                # 适配点：Peewee 包装 psycopg2 异常后，真实 SQLSTATE 可能藏在
```

Ruff (0.16.0)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)